pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -93,6 +93,8 @@ type PostgresSpec struct {
 	// deprecated json tags
 	InitContainersOld       []v1.Container `json:"init_containers,omitempty"`
 	PodPriorityClassNameOld string         `json:"pod_priority_class_name,omitempty"`
+
+	AdditionalTopologySpreadConstraints []v1.TopologySpreadConstraint `json:"additionalTopologySpreadConstraints,omitempty"`
```

Comment:
can you move it behind Tolerations? That would fit together, I believe. Also drop the `Additional` word, because so far we do not define any of these constraints. This will be the only place, where one can define it.